js_modules/dagit/packages/core/src/runs/RunTimeline.tsx

```diff
@@ -305,16 +305,18 @@ const RunTimelineRow = (props: RowProps) => {
   const batched = React.useMemo(() => {
     const batches: RunBatch[] = runs
       .map((run) => {
-        const left = Math.floor(((run.startTime - start) / rangeLength) * width);
+        const startTime = run.startTime;
+        const endTime = run.endTime || Date.now();
```

Comment:
Ah, also fixed this issue where timeline chunks for in-progress runs weren't being stretched to "now".